purchase_stock_price_variance/models/res_company.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2025 Quartile (https://www.quartile.co)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    price_variance_threshold = fields.Boolean(
+        string="Enable price variance threshold check"
+    )
+    price_variance_threshold_percent = fields.Float(
+        help="Maximum variance (in percent) allowable between the product's standard price"
+        " and purchase receipt unit price."
+    )
+    price_variance_threshold_amount = fields.Float(
+        help="Maximum allowable variance (in monetary amount, based on company currency)"
+        " between the product's standard price and the purchase receipt unit price."
+    )
```

Comment:
Let's make sure that the values are positive here.
```suggestion
    )

    @api.constrains("price_variance_threshold_percent", "price_variance_threshold_amount")
    def _check_price_variance_threshold(self):
        for rec in self:
            if rec.price_variance_threshold_percent < 0 or rec.price_variance_threshold_amount < 0:
                raise ValidationError(_("The threshold values cannot be negative."))
```

Reply:
I have added this section